packages/m5flowui/v1.4.0/generic/stubs

```diff
@@ -0,0 +1 @@
+../../../../tools/stubber/stubs/M5_FlowUI_v1_4_0-beta/
```

Comment:
@Josverl,

Are you okay with me symlinking to [micropython-stubber](https://github.com/Josverl/micropython-stubber/tree/master/stubs/M5_FlowUI_v1_4_0-beta) via a submodule? Previously I had just been regenerating the stubs myself, as I didn't want to just copy/paste them. However, I've been struggling with the M5 FlowUI (is it even suppose to run on a [standard esp32 dev board](http://www.hiletgo.com/ProductDetail/1906566.html#)?) and figured doing it this way is probably more appropriate anyways. Also went ahead and [appended your license](db9323bbf10eb23cd119d737b3c9b11e652f76c6) as micropython-stubber is the core of this repo.

Reply:
Also, as I stated in the PR body, the whole M5 Stack thing is confusing me haha.

Is [m5stack/UIFlow-Code](https://github.com/m5stack/UIFlow-Code) where you sourced the firmware from? That's all I could really find for sourcing frozen modules, but it obviously does not contain all of them. Are the rest of the ones you stubbed from micropython v1.11? Thanks.